.github/workflows/snap.yaml

```diff
@@ -33,14 +33,19 @@ on:
         required: false
         type: boolean
       test-script:
-        description: "Custom script to run some custom snap testing"
+        description: "custom script to run some custom snap testing"
```

Comment:
?

Reply:
All the other description starts with lower case.
So I decided to harmonize.
Not a single description has a period so no uppercase sounds good.